lib/compiler/plugins-loader.ts

```diff
@@ -31,7 +30,9 @@ export interface MultiNestCompilerPlugins {
 export class PluginsLoader {
   public load(plugins: PluginEntry[] = []): MultiNestCompilerPlugins {
     const pluginNames = plugins.map((entry) =>
-      isObject(entry) ? (entry as PluginAndOptions).name : (entry as string),
+      entry !== null && typeof entry === 'object'
+        ? (entry as PluginAndOptions).name
+        : (entry as string),
```

Comment:
```suggestion
      entry ? (entry as PluginAndOptions)?.name : (entry as string)
```
?

Reply:
@Tony133 :ping_pong:

Reply:
@Tony133  :ping_pong:

Reply:
Oops, I must have missed the notification for this PR 😅, sorry  🙏🏻